<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MEGAROBO</string>
  </property>
  <widget class="QTabWidget" name="centralWidget">
   <property name="tabBarAutoHide">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File(&amp;F)</string>
    </property>
    <addaction name="actionPoweroff"/>
    <addaction name="actionReboot"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuOperate">
    <property name="title">
     <string>Operate(&amp;O)</string>
    </property>
    <addaction name="actionStop"/>
    <addaction name="separator"/>
    <addaction name="actionUpload"/>
    <addaction name="actionDownload"/>
    <addaction name="actionStore"/>
    <addaction name="actionSync"/>
    <addaction name="separator"/>
    <addaction name="actionSearch"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help(&amp;H)</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionRead_me"/>
    <addaction name="actionErrant"/>
    <addaction name="actionFAQ"/>
   </widget>
   <widget class="QMenu" name="menuToo">
    <property name="title">
     <string>Tool(&amp;T)</string>
    </property>
    <addaction name="actionWifi"/>
    <addaction name="actionPref"/>
    <addaction name="actionChange_Password"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View(&amp;V)</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuOperate"/>
   <addaction name="menuToo"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>48</width>
     <height>36</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <property name="floatable">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionStop">
   <property name="icon">
    <iconset resource="../../res/res.qrc">
     <normaloff>:/res/image/icon/stop.png</normaloff>:/res/image/icon/stop.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Stop</string>
   </property>
   <property name="toolTip">
    <string>Emergency Stop</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>&amp;Help</string>
   </property>
  </action>
  <action name="actionUpload">
   <property name="icon">
    <iconset resource="../../res/res.qrc">
     <normaloff>:/res/image/icon/upload.png</normaloff>:/res/image/icon/upload.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Upload</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+U</string>
   </property>
  </action>
  <action name="actionDownload">
   <property name="icon">
    <iconset resource="../../res/res.qrc">
     <normaloff>:/res/image/icon/download.png</normaloff>:/res/image/icon/download.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Download</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionStore">
   <property name="icon">
    <iconset resource="../../res/res.qrc">
     <normaloff>:/res/image/icon/store.png</normaloff>:/res/image/icon/store.png</iconset>
   </property>
   <property name="text">
    <string>S&amp;tore</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="actionSync">
   <property name="icon">
    <iconset resource="../../res/res.qrc">
     <normaloff>:/res/image/icon/sync.png</normaloff>:/res/image/icon/sync.png</iconset>
   </property>
   <property name="text">
    <string>S&amp;ync</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actionChinese">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Chinese</string>
   </property>
  </action>
  <action name="actionEnglish">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;English</string>
   </property>
  </action>
  <action name="actionMega">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;MEGAROBO</string>
   </property>
  </action>
  <action name="actionClassic">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Classic</string>
   </property>
  </action>
  <action name="actionSearch">
   <property name="icon">
    <iconset resource="../../res/res.qrc">
     <normaloff>:/res/image/icon/find.png</normaloff>:/res/image/icon/find.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Search</string>
   </property>
   <property name="toolTip">
    <string>SearchDevice</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionConnect">
   <property name="icon">
    <iconset resource="../../res/res.qrc">
     <normaloff>:/res/image/h2product/disconnect.png</normaloff>:/res/image/h2product/disconnect.png</iconset>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
   <property name="toolTip">
    <string>Connect</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionIP">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>MRX-H2:[0.0.0.0]</string>
   </property>
  </action>
  <action name="actionReboot">
   <property name="text">
    <string>&amp;Reboot</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+R</string>
   </property>
  </action>
  <action name="actionPoweroff">
   <property name="text">
    <string>&amp;Poweroff</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+P</string>
   </property>
  </action>
  <action name="actionWifi">
   <property name="text">
    <string>&amp;Wifi</string>
   </property>
  </action>
  <action name="actiontest">
   <property name="text">
    <string>test</string>
   </property>
  </action>
  <action name="actionPref">
   <property name="icon">
    <iconset resource="../../res/res.qrc">
     <normaloff>:/res/image/icon/205.png</normaloff>:/res/image/icon/205.png</iconset>
   </property>
   <property name="text">
    <string>Pref</string>
   </property>
  </action>
  <action name="actionErrant">
   <property name="text">
    <string>Errant...</string>
   </property>
  </action>
  <action name="actionFAQ">
   <property name="text">
    <string>FAQ...</string>
   </property>
  </action>
  <action name="actionRead_me">
   <property name="text">
    <string>Read me...</string>
   </property>
  </action>
  <action name="actionChange_Password">
   <property name="icon">
    <iconset resource="../../res/res.qrc">
     <normaloff>:/res/image/icon/51.png</normaloff>:/res/image/icon/51.png</iconset>
   </property>
   <property name="text">
    <string>Change Password</string>
   </property>
   <property name="toolTip">
    <string>Change administrator password</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../../res/res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>599</x>
     <y>449</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
